gapic-generator/templates/default/service/client/_client.erb

```diff
@@ -94,8 +94,7 @@ class <%= service.client_name %>
     <%- unless service.generic_endpoint? -%>
     # Use self-signed JWT if the scope and endpoint are unchanged from default,
```

Comment:
The change in the code was not reflected in the comments directly above.

Reply:
Good catch on that. @bshaffer please update

Reply:
Done!